src/components/NavBar.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react'
+import Styled from 'styled-components'
+import BannerPromo from './molecules/BannerPromo'
+import NavButtons from './molecules/NavButtons'
+import { StaticQuery, graphql } from 'gatsby'
+
+/** Overview of NavBar component:
+ *  - Created a styled div NavContainer which wraps our NavBar
+ *  - NavBar renders NavButtons and Banner Promo
+ *  - NavButtons and Banner Promo will be passed down props such as icons and text that are queried dynamically from contentful
+ *
+ */
+//Another comment
```

Comment:
??

Reply:
We had that random comment when we were trying to change the name of the file but git config was ignoring it.

Reply:
We'll remove it